contracts/src/Teleporter/ReceiptQueue.sol

```diff
@@ -5,18 +5,26 @@
 
 pragma solidity 0.8.18;
 
-import "./ITeleporterMessenger.sol";
+import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
```

Comment:
I'm not opposed to enabling the `no-global-imports` linting rule and correcting everywhere needed now. It will put us in the best position moving forward I think.

Reply:
Updated now. I think having different linting rules for different directories would likely cause more confusion that benefits.